platform/osx/detect.py

```diff
@@ -199,4 +202,19 @@ def configure(env):
         env.Append(CPPDEFINES=["VULKAN_ENABLED"])
         env.Append(LINKFLAGS=["-framework", "Metal", "-framework", "QuartzCore", "-framework", "IOSurface"])
         if not env["use_volk"]:
-            env.Append(LINKFLAGS=["-L$VULKAN_SDK_PATH/MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/", "-lMoltenVK"])
+            env.Append(LINKFLAGS=["-lMoltenVK"])
+            mvk_found = False
+            if env["VULKAN_SDK_PATH"] != "":
+                mvk_path = os.path.join(
+                    os.path.expanduser(env["VULKAN_SDK_PATH"]), "MoltenVK/MoltenVK.xcframework/macos-arm64_x86_64/"
+                )
+                if os.path.isfile(os.path.join(mvk_path, "libMoltenVK.a")):
+                    mvk_found = True
+                    env.Append(LINKFLAGS=["-L" + mvk_path])
+            if not mvk_found:
+                mvk_path = get_mvk_sdk_path()
+                if os.path.isfile(os.path.join(mvk_path, "libMoltenVK.a")):
+                    mvk_found = True
+                    env.Append(LINKFLAGS=["-L" + mvk_path])
+            if not mvk_found:
+                print("MoltenVK SDK installation directory not found, use VULKAN_SDK_PATH to specify SDK path.")
```

Comment:
This should probably abort, as otherwise the print will be at the top of the build logs and might easily be missed. What users will see is the linker complaining that `-lMoltenVK` can't be found.

Reply:
Done.